src/components/jacobian_component/jacobian.sh

```diff
@@ -44,12 +55,21 @@ setup_jacobian() {
     sed -i -e "s:{RunName}:${RunName}:g" \
            -e "s:{InversionPath}:${InversionPath}:g" jacobian_runs/run_prior_simulation.sh
 
+    # Modify HEMCO_Config.rc to turn off individual emission inventories
+    # and use total emissions saved out from prior emissions simulation
+    # instead
+    # Do this in template run directory to avoid having to repeat for each
+    # Jacobian run directory
+    printf "\nTurning on use of total prior emissions in HEMCO_Config.rc.\n"
+    sed -i -e "s|UseTotalPriorEmis      :       false|UseTotalPriorEmis      :       true|g" \
+           -e "s|AnalyticalInversion    :       false|AnalyticalInversion    :       true|g" \
+           -e "s|GFED                   : on|GFED                   : off|g" ${RunTemplate}/HEMCO_Config.rc
```

Comment:
I would think that setting `UseTotalPriorEmis` true would prevent GFED from being used anyways?

Reply:
Unfortunately, it does not because GFED is a HEMCO extension. We can use `UseTotalPriorEmis` in HEMCO_Config.rc to block off the emissions data read for GFED emissions, but that does not prevent HEMCO from calling the GFED extension which will then look for the (now missing) data. Therefore, we have to also set the GFED extension to off.